app/main.py

```diff
@@ -211,4 +18,5 @@ def battle(knightsConfig):
     }
 
 
-print(battle(KNIGHTS))
+if __name__ == '__main__':
+    print(battle(KNIGHTS))
```

Comment:
Do you need this?

Reply:
if __name__ == '__main__' This is the entry point. If I want to change or add data, I can run this function here (+import data) sequentially with different data.
About the print() I agree. You don't need it here.

Reply:
`def battle(knightsConfig):` and pls use `snake_case` -> `knights_config` 

in main on str 6

Reply:
Tnx for camel Case)